pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.447</version>
+    <version>2.30</version>
```

Comment:
Would be better to pick 2.35 now. There were some issues with Restricted checks in 2.30